### PR TITLE
Mp closing footer styles

### PR DESCRIPTION
**Pull request by prechter**
[Surge site](http://bloody-waves.surge.sh/)

The last of the styling pull requests (I hope). Please let me know if there's any glaring issues or polish I could add.

| Desktop | Mobile |
| ------------ | ------------- |
| screen shot 2019-02-21 at 2 54 36 pm | screen shot 2019-02-21 at 2 54 14 pm |

### CODE REVIEW

File: src/stylesheets/blocks/_closing.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  .sm-sans-serif-text {
+    color: color(emerald-green);
+    margin-bottom: rem(25);
+
+    @media (screen(lg)) {
+      text-align: left;
+    }
+  }
+
+  .closing-section__figure {
```
Comment (jeremyfrank) [inline src/stylesheets/blocks/_closing.scss:20]: I would un-nest the figure class.

File: src/stylesheets/blocks/_closing.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    display: none;
+
+    @media (screen(lg)) {
+      display: inline;
+      float: right;
+
+      .closing-section__figure__img {
```
Comment (jeremyfrank) [inline src/stylesheets/blocks/_closing.scss:27]: I would also un-nest the image class.

File: src/stylesheets/blocks/_concept.scss [hunk 3]
```diff
@@ -90,6 +89,18 @@
     display: none;
   }
 
+  .concept-section__list__item__text {
```
Comment (jeremyfrank) [inline src/stylesheets/blocks/_concept.scss:92]: Un-nest this selector.

File: src/stylesheets/blocks/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    .lg-text {
+      font-size: rem(14);
+      line-height: rem(24);
+
+      @media (screen(lg)) {
+        font-size: rem(15);
+      }
+    }
+
+    .footer__logo {
```
Comment (jeremyfrank) [inline src/stylesheets/blocks/_footer.scss:39]: Un-nest.

File: src/stylesheets/blocks/_footer.scss [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  display: flex;
+  font-size: rem(17);
+  height: rem(45);
+  justify-content: center;
+  letter-spacing: 0.05em;
+  margin: rem(30) auto rem(10) auto;
+  text-decoration: none;
+  text-transform: uppercase;
+  transition-duration: 0.1s;
+  transition-timing-function: ease-in-out;
```
Comment (jeremyfrank) [inline src/stylesheets/blocks/_footer.scss:89]: These two can be combined into the shorthand: `transition: 0.1s ease-in-out;`. Also you don't need to set this rule (and potentially all these other rules) for visited and hover together. You should be able to break them out like this:
```
.footer__btn {
  [common styles here, for example...]
  text-transform: uppercase;
  transition: 0.1s ease-in-out;
  width: rem(230);

  &:hover {
    [hover specific styles here (no transition necessary)]
  }

  &:visited {
    [visited specific styles here, if necessary (no transition necessary)]
  }
}
```

File: src/stylesheets/utility/_headings.scss [hunk 6]
```diff
@@ -44,3 +44,18 @@
     }
   }
 }
+
+.lg-title {
+  color: color(emerald-green);
+  font-family: $serif;
+  font-size: rem(30);
+  font-weight: lighter;
```
Comment (jeremyfrank) [inline src/stylesheets/utility/_headings.scss:52]: My initial hunch was that font-weight keywords other than normal and bold, might not have consistent browser support, so I did a search. Turns out that's not correct, BUT keyword values are dependent on what's available in the font family. https://css-tricks.com/almanac/properties/f/font-weight/#article-header-id-0 Just something to keep in mind with future work.

**Review by jeremyfrank — APPROVED** [review]
Looks good. I just had some notes in there about un-nesting some of the BEM style classes, that way your selector is shorter and has a lower specificity.

File: src/stylesheets/utility/_headings.scss (hunk 6 above)
Comment (prechter) [reply src/stylesheets/utility/_headings.scss:52]: Oh interesting. After playing with it some more there really is no `lighter` for this font, just `normal` or `bold` so I kept `normal`.

**Pull request merged**